ws-nextjs-app/pages/ws/languages.page.tsx

```diff
@@ -42,6 +43,8 @@ export const getServerSideProps: GetServerSideProps = async context => {
     };
   }
 
+  const toggles = await getToggles('ws');
```

Comment:
I don't think you need this - toggles are being fetched within `getPageData` & are cached and available where they are needed. Let's remove.